src/promptflow-evals/promptflow/evals/evaluate/_utils.py

```diff
@@ -11,8 +12,14 @@
 
 import pandas as pd
 
+from azure.ai.ml import MLClient

```

Comment:
I know getting rid of the MLClient dependency will be the next step. For now, if the change isn't big, can we lazy import MLClient when it's needed?

Reply:
Fixed.